Assets/HoloToolkit/Input/Scripts/InputEventData/SourcePositionEventData.cs

```diff
@@ -14,16 +14,18 @@ public class SourcePositionEventData : BaseInputEventData
         /// <summary>
         /// The new position of the source.
         /// </summary>
-        public Vector3 Position { get; private set; }
+        public Vector3 PointerPosition { get; private set; }
+        public Vector3 GripPosition { get; private set; }
 
         public SourcePositionEventData(EventSystem eventSystem) : base(eventSystem)
         {
         }
 
-        public void Initialize(IInputSource inputSource, uint sourceId, object tag, Vector3 position)
+        public void Initialize(IInputSource inputSource, uint sourceId, object tag, Vector3 pointerPosition, Vector3 gripPosition)
```

Comment:
Basically this would break someone who is updating to the toolkit for sure. If we can help avoid that then great.